Labs/Alyssa_Giedd_Lab7.ipynb

```diff
@@ -633,6 +779,205 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x130cb4c0f60>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mass_signal_slice, background_slice = slicer(0, 2000, 'mass', signal, background)\n",
+    "ee2_signal_slice, background_slice = slicer(0, 2000, 'angularity', signal, background)\n",
```

Comment:
all of your graphs look great to me, although I am not sure if we need to graph all the variables again, since in lab 7 we found the cut of a variable that gave us a good significance